### ISSUE TITLE
Assess "building your future" PDFs

### ISSUE THREAD

**Issue by aprocik1** [department-of-veterans-affairs/va.gov-team#79115]
## Description
First, we need to migrate these legacy _Building Your Future With the GI Bill®_ PDFs:

- [Part one: A guide to choosing your education pathway](https://www.benefits.va.gov/gibill/docs/gibguideseries/chooseyoureducationpathway.pdf)
- [Part two: A guide to understanding your benefits](https://benefits.va.gov/gibill/docs/gibguideseries/chooseyoureducationbenefits.pdf)
- [Part three: A guide to furthering your career](https://www.benefits.va.gov/GIBILL/docs/gibguideseries/FurtherYourCareer.pdf)

Then, we need to remove the links to the legacy PDFs on these modern pages:

- [Post-9/11 GI Bill (Chapter 33)](https://www.va.gov/education/about-gi-bill-benefits/post-9-11/)
- [Montgomery GI Bill Active Duty (MGIB-AD)](https://www.va.gov/education/about-gi-bill-benefits/montgomery-active-duty/)
- [Montgomery GI Bill Selected Reserve (MGIB-SR)](https://www.va.gov/education/about-gi-bill-benefits/montgomery-selected-reserve/)

### Step 1: Assess legacy PDFs
- [x] DB: Assess the legacy PDF to determine what content still needs to be migrated
- [x] DB: Recommend where the legacy content should be migrated on the modern site
- [x] AP, BP: Review recommendation

### Step 2: Update modern pages
- [x] AP: After stakeholders confirm PDFs have been added to the outreach hub, remove links to them on VA.gov pages
- [x] AP: Share updated pages with MZ
- [ ] MZ: Copyedit and publish

## Criteria
- [x] Content on the legacy PDFs have fully migrated to the modern site
- [ ] Legacy PDFs have been added to the outreach hub
- [ ] Links to the legacy PDFs are removed from the modern site

**Comment by d-beyer** [comment]
@aprocik1 I've reviewed the PDFs and I don't think there's any information that we actually have to migrate. However, the education stakeholders commented on the audit that they would like to retain these for now. I think the simplest approach would just be to rehost them in the outreach hub and update the links to them on the modern site. What do you think?

**Comment by aprocik1** [comment]
@d-beyer, that aligns with my assessment also. I don't think this information is missing on VA.gov.

However, I think the way it's presented in PDF 1 could be helpful for Veterans (types of degrees / schools / learning). For that reason, we could make an argument for creating a new resources and support page like this:

<details>
<summary>Scratchpad</summary>

- [H1] What to consider when pursuing higher education
- [H2] Types of GI Bill benefits (inspired by PDF 2, page 2)
     - Link to [About GI Bill benefits](https://www.va.gov/education/about-gi-bill-benefits/)
- [H2] Types of higher education opportunities (inspired by PDF 1, pages 4-6)
     - Link to [How to use your GI Bill benefits](https://www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/)
     - Link to [GI Bill Comparison Tool](https://www.va.gov/education/gi-bill-comparison-tool/)
- [H2] Additional resources (inspired by PDF 1, pages 7-8 and PDF 2, page 8)
     - Link to [Veteran Readiness and Employment (Chapter 31)](https://www.va.gov/careers-employment/vocational-rehabilitation/)
     - Link to [Educational and career counseling (VA Chapter 36)](https://www.va.gov/careers-employment/education-and-career-counseling/)
     - Link to [VetSuccess on Campus (VSOC)](https://www.va.gov/careers-employment/vetsuccess-on-campus/)

</details>

I'm not sure if a new page like this would add more clarity or cause more confusion. What do you think?

Regardless of that decision, I think we should ask SMEs to confirm that the PDFs are still accurate before we add them to the outreach hub. For example, they may want to remove mentions of VET TEC, since the program has ended.

**Comment by bethpottsVADEPO** [comment]
@d-beyer @aprocik1 Thanks for digging into this. The idea for the new R&S page is interesting and something that Danielle and Mikki will consider in the course of streamlining the hubs. It will be better to fold this work into streamlining rather than adding another task to migration. Thanks for the proposal!

Short term, I'll ask SMEs or VBA comms to add these PDFs to the outreach hub and you all can remove the links to the PDFs from these pages (I'll tell you when to go ahead with that; for now, please hold):
[Post-9/11 GI Bill (Chapter 33)](https://www.va.gov/education/about-gi-bill-benefits/post-9-11/)
[Montgomery GI Bill Active Duty (MGIB-AD)](https://www.va.gov/education/about-gi-bill-benefits/montgomery-active-duty/)
[Montgomery GI Bill Selected Reserve (MGIB-SR)](https://www.va.gov/education/about-gi-bill-benefits/montgomery-selected-reserve/)

We're trying to get away from linking to PDFs, that's why we wouldn't want to update those links that are on the 3 program pages, but instead we'll remove them. 

**Next:**
- I'll have a conversation with Tammy about this and will ask her to work with VBA comms to add the 3 PDFs to the outreach hub after Education Service reviews them for timeliness and accuracy. Thanks for pointing that out, specifically about VET TEC.

**Comment by aprocik1** [comment]
Update: Beth and I discussed this in a meeting.

Next steps: First, education stakeholders are going to add these PDFs to the outreach hub. Then, we can remove links to these PDFs from VA.gov pages.

**Comment by bethpottsVADEPO** [comment]
@aprocik1 I just heard back from Terry Warren in Education Service. He gave us the green light to remove the links to these PDFs from our pages even though they aren't in the Outreach Hub yet. Thank you for moving forward with that. 

We'll want to hold on asking Education Service to redirect the PDFs until they've uploaded them to the Outreach Hub. I'll circle back with you when that has happened. So for now, Jenny should wait on communicating about these PDFs in a redirect email.

**Comment by aprocik1** [comment]
@megzehn and @saratorres2, can you copyedit and publish the changes on these pages?
- [Post-9/11 GI Bill (Chapter 33)](https://preview-prod.vfs.va.gov/preview?nodeId=935)
- [Montgomery GI Bill Active Duty (MGIB-AD)](https://preview-prod.vfs.va.gov/preview?nodeId=932)
- [Montgomery GI Bill Selected Reserve (MGIB-SR)](https://preview-prod.vfs.va.gov/preview?nodeId=934)

On each page, I removed the final Q&A: How can I get more information? I know Randi has draft edits on the two MGIB pages for deprecating WAVE. I'm not sure when those should be published. Thanks for your support on this!

@d-beyer, these pages aren't ready for Jenny's redirect. Beth will let us know when we can move forward on that.

**Comment by saratorres2** [comment]
@aprocik1 I reviewed the [Montgomery GI Bill Active Duty (MGIB-AD)](https://preview-prod.vfs.va.gov/preview?nodeId=932) page with @megzehn and published changes.

@RLHecht said we should [hold off on publishing changes](https://dsva.slack.com/archives/C03MP8MEPCN/p1719345216576909?thread_ts=1719344806.763009&cid=C03MP8MEPCN) to the other two pages until we publish the WAVE deprecation changes:
[Montgomery GI Bill Selected Reserve (MGIB-SR)](https://preview-prod.vfs.va.gov/preview?nodeId=934)
[Montgomery GI Bill Active Duty (MGIB-AD)](https://preview-prod.vfs.va.gov/preview?nodeId=932)

**Comment by aprocik1** [comment]
Update: All of these pages have been updated and published:

- [Post-9/11 GI Bill (Chapter 33)](https://preview-prod.vfs.va.gov/preview?nodeId=935)
- [Montgomery GI Bill Active Duty (MGIB-AD)](https://preview-prod.vfs.va.gov/preview?nodeId=932)
- [Montgomery GI Bill Selected Reserve (MGIB-SR)](https://preview-prod.vfs.va.gov/preview?nodeId=934)

I'm closing and tracking this ticket as complete.
